hack/deploy-provider.sh

```diff
@@ -48,6 +48,9 @@ spec:
           containers:
           - name: package-runtime
             image: ${BUILD_REGISTRY}/${PROJECT_NAME}-${ARCH}
+            env:
+              - name: RECREATE_MISSING_BUCKET
+                value: "${RECREATE_MISSING_BUCKET}"
```

Comment:
Why are we passing in this env? :thinking:

Reply:
@nolancon because the cli flag is a boolean flag, so it is difficult to add it, while env variable should be empty without any extra logic.